app/controllers/support_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SupportController < LoggedAreaController
+  skip_before_action :verify_authenticity_token
```

Comment:
I spent some time trying to avoid skipping this verification, but I wasn't able to. Without it, the error is that the token is invalid. Poss caused by caching?